ios/PurchasesHybridCommon/PurchasesHybridCommonTests/CustomerInfo+TestExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CustomerInfo+TestExtensions.swift
+//  PurchasesHybridCommonTests
+//
+//  Created by Will Taylor on 5/27/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import RevenueCat
+
+extension CustomerInfo {
+    static func fromJSON(_ jsonString: String) -> CustomerInfo? {
+        let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
+        decoder.dateDecodingStrategy = .iso8601
+
+        guard let data = jsonString.data(using: .utf8) else { return nil }
```

Comment:
You can do `Data(jsonString.utf8)` and that won't return an optional.

Reply:
Oh, nice! I've used that initializer and removed the `guard let`s where this helper is being called.